packages/flutter_tools/lib/src/commands/run.dart

```diff
@@ -31,7 +33,7 @@ abstract class RunCommandBase extends FlutterCommand {
           'Note: this flag will be removed in favor of the --debug/--profile/--release flags.');
 
     argParser.addFlag('trace-startup',
-        negatable: true,
+        negatable: false,
```

Comment:
Yeah, but more terse for the common case (flutter run). I'll revert this, we can revisit.